catalog/ui/catalog-ui-search/package.json

```diff
@@ -5,11 +5,13 @@
   "version": "0.0.1",
   "license": "LGPL-3.0",
   "scripts": {
-    "startplus": "grunt startplus --webpackConfig=./webpack/config/dev.js",
```

Comment:
Looking through the list, we no longer require these dependencies:

- compression
- connect-livereload
- cors
- grunt
- grunt-cli
- grunt-contrib-clean
- grunt-contrib-cssmin
- grunt-contrib-jshint
- grunt-contrib-less
- grunt-contrib-watch
- grunt-express-server
- grunt-open
- grunt-webpack
- http-proxy
- load-grunt-tasks
- morgan